core/src/main/scala/com/nvidia/spark/rapids/tool/planparser/SQLPlanParser.scala

```diff
@@ -313,7 +315,7 @@ object ExecInfo {
       duration,
       nodeId,
       opType,
-      isSupported,
+      finalSupported,
       children,
       stages,
       shouldRemove,
```

Comment:
Do we need to make changes to the usage of `containsUDF` when passing to `createExecNoNode()`

Reply:
I don't think so. Now `containsUDF` denotes that the exec contains Scala UDFs after we changed the matching regex, so for python UDFs, this value would remain `false` as intended.